dsl/src/main/groovy/org/cristalise/dsl/persistency/outcome/Attribute.groovy

```diff
@@ -45,6 +45,9 @@ class Attribute {
 
     boolean required = false
 
+    Integer precision = null //mapped to xs:totalDigits
+    Integer scale = null     //mapped to xs:fractionDigits
```

Comment:
Why not use the same names as the `xs:` restrictions? The problem with using `scale` is that usually (like for instance in the implementation of BigDecimal) the scale can be negative, while `fractionDigits` cannot.